src/components/Menu/config/config.ts

```diff
@@ -52,14 +52,22 @@ const config: (t: ContextApi['t']) => ConfigMenuItemsType[] = (t) => [
   },
```

Comment:
/config/config.ts ?

Reply:
This wasn't used anywhere so I removed it. It might have been leftover from the rebase.

Reply:
Yeah maybe we should change it. It is what is on develop right now.

Reply:
In this folder there are 2 config file, one for the menu and one for the footer.
Maybe we should rename this one to menuConfig then ?